bundles/org.openhab.binding.homeconnect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,676 @@
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color68"> Color 68</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color69"> Color 69</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color70"> Color 70</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color71"> Color 71</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color72"> Color 72</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color73"> Color 73</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color74"> Color 74</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color75"> Color 75</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color76"> Color 76</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color77"> Color 77</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color78"> Color 78</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color79"> Color 79</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color80"> Color 80</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color81"> Color 81</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color82"> Color 82</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color83"> Color 83</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color84"> Color 84</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color85"> Color 85</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color86"> Color 86</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color87"> Color 87</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color88"> Color 88</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color89"> Color 89</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color90"> Color 90</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color91"> Color 91</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color92"> Color 92</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color93"> Color 93</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color94"> Color 94</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color95"> Color 95</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color96"> Color 96</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color97"> Color 97</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color98"> Color 98</option>
+				<option value="BSH.Common.EnumType.AmbientLightColor.Color99"> Color 99</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="ambient_light_custom_color_state">
+		<item-type>Color</item-type>
+		<label>Ambient light custom color</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Any suggestion? I'm not sure what to choose instead.

Reply:
E.g. `Ambient Light (Custom)`